Package.resolved

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
@liuxuan30 This is the .resolved for working on the SwiftPM version of Charts.

The one in `/xcsharedata/swiftpm` is for working on the .framework version used by Carthage and Cocoapods.